### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by EvsenevDev**
- Changed support email
- [Fix] Bug in which spaces don't appear when typing
- Added support for sharing a single receipt image

### CODE REVIEW

File: SmartReceipts/Common/Constants/Constants.m [hunk 1]
```diff
@@ -30,6 +30,6 @@ void SRDelayedExecution(NSTimeInterval seconds, ActionBlock action) {
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, seconds * NSEC_PER_SEC), dispatch_get_main_queue(), action);
 }
 
-NSString *const FeedbackEmailAddress =              @"will.r.baum" @"ann@gm" @"ail.com";
+NSString *const FeedbackEmailAddress =              @"supp" @"port" @"@sma" @"rtreceipts.co";
```
Comment (wbaumann) [inline SmartReceipts/Common/Constants/Constants.m]: I think this is misspelled. It's currently

- suppport@smartreceipts.co

But should be:

- - support@smartreceipts.co

One too many 'p' letters :)

File: SmartReceipts/Common/Utils/EurekaWhitespaceWorkaround.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import Foundation
+import Eureka
+
+final class EurekaWhitespaceWorkaround {
+    
+    static func configureTextCells() {
+        let workaround = EurekaWhitespaceWorkaround()
+        TextRow.defaultCellSetup = { cell, row in
+            // the workaround object is retained here on purpose
```
Comment (wbaumann) [inline SmartReceipts/Common/Utils/EurekaWhitespaceWorkaround.swift]: Want to add another comment, which points to that issue link? Thanks!

File: SmartReceipts/Common/Utils/EurekaWhitespaceWorkaround.swift [hunk 3]
```diff
@@ -0,0 +1,36 @@
+//
+//  EurekaWhitespaceWorkaround.swift
+//  SmartReceipts
+//
+//  Created by Bogdan Evsenev on 13/11/2018.
+//  Copyright © 2018 Will Baumann. All rights reserved.
```
Comment (wbaumann) [inline SmartReceipts/Common/Utils/EurekaWhitespaceWorkaround.swift]: Or up here for the link to the Eureka issue, just so we can track here

File: SmartReceipts/Common/Constants/Constants.m (hunk 1 above)
Comment (EvsenevDev) [reply SmartReceipts/Common/Constants/Constants.m]: @wbaumann yes, sorry 👍

File: SmartReceipts/Common/Utils/EurekaWhitespaceWorkaround.swift (hunk 3 above)
Comment (EvsenevDev) [reply SmartReceipts/Common/Utils/EurekaWhitespaceWorkaround.swift]: @wbaumann makes sense, i'll add it to this line

**Pull request merged**